scripts/build.sh

```diff
@@ -46,12 +46,3 @@ export GOOS=linux
 export GOARCH=amd64
 mkdir -p "${build_dir}/${GOOS}/x86_64"
 "${go_build[@]}" -o "${build_dir}/${GOOS}/x86_64/${plugin_name}" . || exit 1
-
```

Comment:
We need to preserve this custom section, please configure this file to be ignored in .sync.yml:

```
scripts/build.sh:
  unmanaged: true
```